docs/source/reference/config.rst

```diff
@@ -40,6 +40,31 @@ Available fields and semantics:
     - gcp
     - kubernetes
 
+  docker:
+    # Additional Docker run options (optional).
+    #
+    # When image_id: docker:<docker_image> is used in a task YAML, additional
+    # run options for starting the Docker container can be specified here.
+    # These options will be passed directly as command line args to `docker run`,
+    # see: https://docs.docker.com/reference/cli/docker/container/run/
+    #
+    # The following run options are applied by default and cannot be overridden:
```

Comment:
Where have we guarantee this "cannot be override"? In the code it seems like we are just appending them in the docker run cmds and follow the default behaviour of multiple same argument to the command. Do we need some checks on the docker run options to make sure it does not conflict w/ existing options?

Reply:
It is hard to check the arguments for docker. It might be fine for now to allow the program to proceed and let it error out after the cluster is launched. Since we have allowed the change of `run_options` for an existing cluster, a user can quickly fix it according to the error message. Wdyt?

Reply:
sounds good!